src/layouts/Root/components/Header.tsx

```diff
@@ -80,14 +80,14 @@ const Logo: React.FC = () => (
           css={css`
             width: ${p => p.theme.iconSizes.lg};
             height: ${p => p.theme.iconSizes.lg};
-            color: ${PALETTE.green[400]};
+            color: ${p => getColorV8('green', 400, p.theme)};
```

Comment:
```suggestion
            color: ${p => getColor({ theme: p.theme, hue: 'green', shade: 500 })};
```

Reply:
Although this PR updated the Garden logo under some `example` files, I'm reticent to change it here because v8 `green[400]` was blessed by design as the Garden product color. Let's leave this until we get future input from design.